atsign-foundation/at_server
# feat: Virtual env changes for apkam demo atsigns

**- What I did**
- in order to test APKAM onboarding flow, need atsigns in virtual env without pkam keys loaded. Made changes in virtual env to load 2 new atsigns without pkam keys loaded. 
**- How I did it**
- in at_demo_data added a new list "apkamAtsigns" which contain two atsigns - srie, sachin
- in virtual env install pkam keys logic, read the apkam atsigns from at_demo_data and perform only cram auth
- logic for existing atsigns remains the same. Made minor changes to method naming/incorrect key name in print statement
- in create_demo_accounts.sh, create the two apkam atsigns
**- How to verify it**
- build secondary binary and copy to tools/build_virtual_environment/ve/contents/atsign/secondary
- docker build from /tools/build_virtual_environment/ve
sudo docker build -t atsigncompany/localnew .
- docker run
sudo docker run  --name at_virtual_env_new -e testingMode="true" -p 6379:6379 -p 25000-25019:25000-25019 -p 64:64 -p 9001:9001 atsigncompany/localnew:latest
The two new atsigns should show up in supervisor status - http://localhost:9001/
